lib/ProductOpener/Tags.pm

```diff
@@ -4613,6 +4613,49 @@ sub add_tags_to_field ($product_ref, $tag_lc, $field, $additional_fields) {
 	return;
 }
 
+sub remove_tag_from_field($product_ref, $tag_lc, $field, $tags_to_remove) {
```

Comment:
You except a list of tag to remove so:
* use plural in function name (remove_tags_from_field)
* your last parameter should be a reference to that list

```suggestion
sub remove_tags_from_field($product_ref, $tag_lc, $field, $tags_to_remove_ref) {
```

you can then get the list with:
```perl
my @tags_to_remove = @$tags_to_remove_ref;
```

Reply:
Here again I misread… you have a string which is a comma separated list of values (maybe it's a good indication that you should document the function ;-) )

So appart from the function name, it's all ok :-)

```suggestion
sub remove_tags_from_field($product_ref, $tag_lc, $field, $tags_to_remove) {
```